modules/katamari/src/test/ts/atomic/api/optional/AssertOptionalTest.ts

```diff
@@ -0,0 +1,47 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+import { Optional } from 'ephox/katamari/api/Main';
+import { assertOptional } from 'ephox/katamari/test/AssertOptional';
+import fc from 'fast-check';
```

Comment:
```suggestion
import * as fc from 'fast-check';
import { Optional } from 'ephox/katamari/api/Optional';
import { assertOptional } from 'ephox/katamari/test/AssertOptional';
```
Just fixing import issues.